src/main/java/Duke.java

```diff
@@ -1,10 +1,42 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke \nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+
+        String userInput;
+        while(in.hasNext()){
+
+            userInput = in.nextLine();
+            if(userInput.equals("bye")){
+                break;
+            }
+            if(userInput.equals("list")){
+                for(int i = 0; i<taskList.size(); i++){
+                    System.out.print(i+1+".");
+                    System.out.println(taskList.get(i).markTask()+taskList.get(i).name);
```

Comment:
for your function .get, perhaps be clearer about what you are getting, such using as getStatus or getName.